app/controllers/application_controller.rb

```diff
@@ -1,33 +1,39 @@
 require './config/environment'
+require 'erubi'
 
 class ApplicationController < Sinatra::Application
   configure do
+    set :erb, escape_html: true
     set :public_folder, 'public'
     set :views, 'app/views'
     logger = Logger.new(File.open("#{root}/../log/#{environment}.log", 'a'))
     logger.level = Logger::DEBUG unless production?
     set :logger, logger
     ActiveRecord::Base.logger = logger
+    set :erb, :escape_html => true
     enable :sessions
     set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
-    # Step 2: Enable Rack::Protection by uncommenting the following lines:
-    #
-    # use Rack::Protection
-    # # `use Rack::Protection` automatically enables all modules except for the
-    # # following, which have to be enabled explicitly
-    # use Rack::Protection::AuthenticityToken
-    # use Rack::Protection::EscapedParams
-    # use Rack::Protection::FormToken
-    # use Rack::Protection::RemoteReferrer
-    #
-    # Implementation Note/Hint:
-    #
-    # You will have to update your forms to include an AuthenticityToken,
-    # per the documentation for Rack::Protection::AuthenticityToken:
-    # https://sinatrarb.com/protection/authenticity_token
+
+    use Rack::Protection
+    # `use Rack::Protection` automatically enables all modules except for the
+    # following, which have to be enabled explicitly
+    use Rack::Protection::AuthenticityToken
+    # use Rack::Protection::EscapedParams # erubi is handling html escaping
```

Comment:
`Rack::Protection::EscapedParams` provides more robust protection because it prevents us from storing malicious data